covid_api/db/static/sites/ny.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "ny",
+  "label": "New York",
+  "center": [ 73.09, 41.0114 ],
```

Comment:
center isn't inside the bounds or polygon, probably -73.09?

Reply:
Good catch. Updated